docs/source/enable_toolkit_selection.rst

```diff
@@ -0,0 +1,134 @@
+.. _toolkit-selection:
+
+Enable Toolkit Selection
+========================
+
+One of the more complicated aspects of Enable is the way in which it selects
+which widget toolkit (Qt, Wx, etc.) to use when constructing a :class:`Window`
+instance. Further, there is also a second choice of which kiva
+:class:`GraphicsContext` implementation to use along with the selected widget
+toolkit.
+
+There are of course defaults for both toolkit and kiva backend, which might even
+depend on the packages available in the Python environment where code is
+running. For most users, these defaults are sufficient and stable over time. For
+users that have specific needs, the widget toolkit and kiva backend can be
+explicitly specified.
+
+
+ETSConfig
```

Comment:
Not a suggested change:
It makes sense to have this section here, but some of this content should also happily sit in the traits docs.  Currently the traits docs (outside of api documentation) have little to no mention of ETSConfig.

Reply:
Ref: https://github.com/enthought/traits/issues/1427
When writing the traits docs in the future it may be useful to reference these docs / vice-versa

Reply:
Agree. Glad there's an issue in traits already.